test/api/models/audit/GenericAuditDetailSpec.scala

```diff
@@ -23,12 +23,12 @@ import support.UnitSpec
 
 class GenericAuditDetailSpec extends UnitSpec {
 
-  val nino: String = "XX751130C"
-  val taxYear: String = "2020-21"
-  val benefitId: String = "a1e8057e-fbbc-47a8-a8b4-78d9f015c253"
-  val userType: String = "Agent"
+  val nino: String                         = "XX751130C"
```

Comment:
vals that are directly assigned a string value don't need the ": String" type declaration